clang-tools-extra/docs/clang-tidy/checks/readability/implicit-bool-conversion.rst

```diff
@@ -147,3 +147,38 @@ Options
       if (foo) {}
       // ^ propose replacement default: if (foo != 0u) {}
       // ^ propose replacement with option `UseUpperCaseLiteralSuffix`: if (foo != 0U) {}
+.. option:: CheckConversionsToBool
+
+   When `true`, the check diagnoses implicit conversions to `bool`. Default is `true`.
```

Comment:
```suggestion
   When `true`, the check diagnoses implicit conversions to ``bool``. Default is `true`.
```

As language construct. Please also follow 80 characters limit. Same for other option.

Reply:
Done Thanks.

Can you please also review ImplicitBoolConversionCheck.cpp?

Reply:
Please wait for other reviewers. I deal only with documentation and minor code style issues.

Reply:
Okay Thanks a lot !